src/extensions/default/MDNDocs/README.md

```diff
@@ -0,0 +1,3 @@
+# Updating the Docs
+* Use the Node script [scrape-mdn](https://github.com/mozilla/brackets/tree/master/src/extensions/extra/MDNDocs/tools/scrape-mdn) to update the `css.json` and `html.json` contents:
```

Comment:
Probably we need to change grunt to avoid to have MDNDocs/tools in the distribution.